components/input.css

```diff
@@ -0,0 +1,84 @@
+@import url('../styles.css');
+
+.inputs{
+    display:flex;
+    flex-direction: column;
+    gap: 20px;
+    align-items: center;
+    justify-content: center;
+    height: max-content;
+    padding: 1rem 4rem;
+    border: 1px solid var(--primary-color);
+    border-radius: 15px;
+}
+
+.inputs input{
+    width: 70%;
+    height: 2rem;
+    padding-left: 1rem;
+    border-radius: 5px;
+}
+
+.input-with-error{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: max-content;
+    padding: 1rem 4rem;
+    border: 1px solid var(--primary-color);
+    border-radius: 15px;
+}
+
+.input-with-error input{
```

Comment:
make changes here too, this can lead to conflicts in the future if suppose inside your .input-with-error class, you have 2 inputs and you want those inputs to have different CSS then this may cause a problem later on .